backend/functions/src/triggers/on-create-comment-on-contract.ts

```diff
@@ -105,8 +104,6 @@ export const onCreateCommentOnContract = functions
     const commentCreator = await getUser(comment.userId)
     if (!commentCreator) throw new Error('Could not find comment creator')
 
-    await addUserToContractFollowers(contract.id, commentCreator.id)
```

Comment:
I think we should send users who commented on the contract, a resolution notification.